code/modules/admin/verbs/onlyone.dm

```diff
@@ -3,12 +3,29 @@
 		alert("The game hasn't started yet!")
 		return
 
+	for(var/mob/living/silicon/S in player_list) //All silicons get made into humans so they can be highlanders, too.
+		if(S.isDead() || !S.client) continue
+		if(is_special_character(S)) continue
+
+		var/mob/living/carbon/human/new_human = new /mob/living/carbon/human(S.loc, delay_ready_dna=1)
+		new_human.setGender(pick(MALE, FEMALE)) //The new human's gender will be random
+		var/datum/preferences/A = new()	//Randomize appearance for the human
+		A.randomize_appearance_for(new_human)
+		new_human.generate_name()
+		new_human.languages |= S.languages
+		if(S.default_language) new_human.default_language = S.default_language
+		if(S.mind)
+			S.mind.transfer_to(new_human)
+		else
+			new_human.key = S.key
+		qdel(S)
+
 	for(var/mob/living/carbon/human/H in player_list)
-		if(H.stat == 2 || !(H.client)) continue
+		if(H.isDead() || !H.client) continue
 		if(is_special_character(H)) continue
 
 		ticker.mode.traitors += H.mind
-		H.mind.special_role = "traitor"
+		H.mind.special_role = "highlander"
```

Comment:
I don't care if other people shat on the floor, you're going to learn to shit in the toilet.

String define.